data/blog/js-sdk-v8-retrospective.md

```diff
@@ -0,0 +1,63 @@
+---
+title: 'Sentry JavaScript SDK v8 - A Retrospective'
+date: '2024-07-16'
+tags: ['javascript', 'sdk']
+draft: false
+summary: 'This post will outline learnings the Sentry SDK team had from releasing v8 of the JavaScript SDKs.'
+images: [/images/js-sdk-v8-retrospective/hero.jpg]
+layout: PostLayout
+canonicalUrl: https://sentry.engineering/blog/js-sdk-v8-retrospective
+authors: ['francesconovy']
+---
+
+On May 13, 2024, the Sentry Web Frontend SDK team shipped [v8.0.0](https://github.com/getsentry/sentry-javascript/releases/tag/8.0.0) of the JavaScript SDKs. This release has been a long time coming - the team has been working on it for multiple months, trying to ensure that all the changes we made in this version were well-tested and that upgrading would be as easy and painless as possible for our users.
+
+However, v8 was quite a massive release, with many changes. The primary change was that `@sentry/node` was rewritten to use OpenTelemetry under the hood. This main change, in turn, required us to make a variety of smaller and larger changes to accommodate the APIs that OpenTelemetry supports. The largest change that was part of this was adjusting all the performance APIs to be compatible with OpenTelemetry.
+
+## Changes between v7 and v8
+
+You can read more about all the changes we did and how to migrate from v7 to v8 either in the [Browser JS Migration docs](https://docs.sentry.io/platforms/javascript/migration/v7-to-v8/) or in the [Node JS Migration docs](https://docs.sentry.io/platforms/javascript/guides/node/migration/v7-to-v8/).
+
+This blog post does not aim to explain all the changes we made in v8. Instead, the goal of this post is to discuss takeaways we identified while and after working on this large release, as well as sharing things we’d do again or differently in future major releases.
```

Comment:
```suggestion
This blog post does not aim to explain all the changes we made in v8. Instead, the goal of this post is to discuss takeaways we identified during and after working on this large release, and more directly, sharing things we’d do again and differently in future major releases.
```

Reply:
> sharing things we’d do again and differently in future major releases.

i was bit confused if you want to share things you will repeat as in yay good,  or if this is all stating what mistakes we made